src/seq/abif/reader.jl

```diff
@@ -0,0 +1,254 @@
+# ABIF Reader
+# ===========
+
+
+export tags, elements
+
+immutable AbifDirEntry
+    name::String
+    number::Int32
+    element_type::Int32
+    element_size::Int32
+    num_elements::Int32
+    data_size::Int32
+    data_offset::Int32
+end
+
+"""
+    AbifReader(input::IO)
+    Create a data reader of the ABIF file format.
+    # Arguments
+    * `input`: data source
+"""
```

Comment:
A minor indent problem here. This will not be displayed as you expect. Please write it like below and check it from your REPL:
```
    AbifReader(input::IO)

Create a data reader of the ABIF file format.

# Arguments

* `input`: data source
```